src/main/java/org/jabref/gui/entryeditor/CitationRelationsTab.java

```diff
@@ -0,0 +1,562 @@
+import org.jabref.logic.l10n.Localization;
+import org.jabref.model.database.BibDatabaseContext;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.StandardField;
+import org.jabref.model.util.FileUpdateMonitor;
+import org.jabref.preferences.JabRefPreferences;
+import org.jabref.preferences.PreferencesService;
+
+import com.tobiasdiez.easybind.EasyBind;
+import org.controlsfx.control.CheckListView;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * GUI for tab displaying an articles citation relations in two lists based on the currently selected BibEntry
+ */
+public class CitationRelationsTab extends EntryEditorTab {
+
+    // Tasks
+    private static BackgroundTask<List<BibEntry>> citingTask;
+    private static BackgroundTask<List<BibEntry>> citedByTask;
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CitationRelationsTab.class);
+    private final EntryEditorPreferences preferences;
+    private final DialogService dialogService;
+    private final BibDatabaseContext databaseContext;
+    private final UndoManager undoManager;
+    private final StateManager stateManager;
+    private final FileUpdateMonitor fileUpdateMonitor;
+    private final PreferencesService preferencesService;
+    private final LibraryTab libraryTab;
+
+    /**
+     * Class to hold a BibEntry and a boolean value whether it's already in the current database or not.
+     */
+    public static class CitationRelationItem {
+        private final BibEntry bibEntry;
+        private final boolean local;
+
+        public CitationRelationItem(BibEntry bibEntry, boolean local) {
+            this.bibEntry = bibEntry;
+            this.local = local;
+        }
+
+        public BibEntry getBibEntry() {
+            return bibEntry;
+        }
+
+        public boolean isLocal() {
+            return local;
+        }
+    }
+
+    public CitationRelationsTab(EntryEditorPreferences preferences, DialogService dialogService, BibDatabaseContext databaseContext, UndoManager undoManager, StateManager stateManager, FileUpdateMonitor fileUpdateMonitor, PreferencesService preferencesService, LibraryTab lTab) {
+        this.preferences = preferences;
+        this.dialogService = dialogService;
+        this.databaseContext = databaseContext;
+        this.undoManager = undoManager;
+        this.stateManager = stateManager;
+        this.fileUpdateMonitor = fileUpdateMonitor;
+        this.preferencesService = preferencesService;
+        this.libraryTab = lTab;
+        setText(Localization.lang("Citation relations"));
+        setTooltip(new Tooltip(Localization.lang("Show articles related by citation")));
+    }
+
+    /**
+     * The Pane that is shown when the functionality is not activated
+     *
+     * @param entry entry that is the context
+     * @return StackPane that is the activation screen
+     */
+    private StackPane getActivationPane(BibEntry entry) {
```

Comment:
Similar to Mr. DLib to be DSGVO conformant. We need a ADR 😅

Reply:
@tobiasdiez we thought there would be a case when a user has already used the API and Added the relations. In that case he maybe dont want the search to be started over and over again when he enters the Tab.

@koppor we will work on this :)

Reply:
> we thought there would be a case when a user has already used the API and Added the relations.

In that case, one would probably has some entries that have citation keys and others don't. So it would be entry-dependent. For now I would remove this additional pane.

@koppor please don't request an ADR for every small implementation detail. An ADR is really to much effort for this here. Instead I would suggest to have ADRs only if there was something controversial during a PR review in order to summarize the outcome of a discussion.

Reply:
I see ADR as "decision records". In case something being unclear at a PR review is IMHO an indicator for a code comment or an ADR. In the concrete case handling GDPR compliance is something which should be documented more prominent than a comment at a PR review or a code comment.

Meanwhile, we are working on a browser-based UI to create and edit ADRs.

Reply:
So should we leave the Panel in, to be DSGVO conformat?

Reply:
We also don't have a confirmation dialog for all the other fetcher, that sends out exactly the same information. For GDPR we can put something in the license information in the installation dialog.